lib/common.h

```diff
@@ -20,13 +20,41 @@
 #ifndef __LIB_COMMON_H__
 #define __LIB_COMMON_H__
 
-/* Get current timestamp in secs. */
+#include <linux/percpu.h>
+#include <linux/time.h>
+
+static DEFINE_PER_CPU(struct timespec64, tfw_cached_ts);
+static DEFINE_PER_CPU(unsigned long, tfw_ts_last_update);
```

Comment:
For consistency's sake, perhaps, `tfw_ts_` prefix for both variables would be cleaner here? `tfw_ts_cached`.